api.eyeofthetiga.com/Guides/CloudAdapter.md

```diff
@@ -16,29 +16,30 @@ add settings defined here https://github.com/Plantiga/CloudAdapter/issues/206
-| 0xD000         | WIFI Service                    | service     |
-| 0xD001         | Network SSID                    | read,write  |
-| 0xD002         | Password                        | write       |
-| 0xD003         | Network List of Available SSIDs | notify      |
-| 0xD004         | Trigger a Connection Attempt    | write       |
-| 0xD005         | Connection Status               | read,notify |
+| `0xD000`       | WIFI Service                    | service     |
+| `0xD001`       | Network SSID                    | read,write  |
+| `0xD002`       | Password                        | write       |
+| `0xD003`       | Network List of Available SSIDs | notify      |
+| `0xD004`       | Trigger a Connection Attempt    | write       |
+| `0xD005`       | Connection Status               | read,notify |
 <br>
 
 ## 0xD000 WIFI Service
 
-If the emulator is started with --no-networks, then no networks will be shown for `0xD003` 
+If the emulator is started with `--no-networks`, then no networks will be shown for `0xD003` 
 network list of available ssids.
 
 Writing the `0xD001` and `0xD002` characteristics will not be enough to start a network connection attempt, `0xD004` should be written to to trigger a network connection attempt. 
 <br>
 
 ## 0xD001 Network SSID
-| Defined Submissions | Description                                                                    |
-| ------------------- | ------------------------------------------------------------------------------ |
-| `"Happy Path"`      | All passwords work on this network, `0xD005` will always notify as `CONNECTED` |
-| `"Bad Password"`    | After `0xD004` is set, then `0xD005` will always notify as `PASSWORD_ERROR`    |
-| `"Bad Network"`     | After `0xD004` is set, then 0xD005 will always notify as `CONNECTION_ERROR`    |
-| `"Firewall"`        | After `0xD004` is set, then 0xD005 will always notify as `NETWORK_ERROR`       |
+| Defined Submissions   | Description                                                                                                              |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `"Happy Path"`        | All passwords work on this network, `0xD005` will always notify as `CONNECTED` after `0xD004` is set                     |
+| `"Bad Network"`       | After `0xD004` is set, then `0xD005` will always notify as `NETWORK_ERROR`                                               |
+| `"Firewall"`          | After `0xD004` is set, then `0xD005` will always notify as `CONNECTION_ERROR`                                            |
+| `"Bad Password"`      | After `0xD004` is set, then `0xD005` will always notify as `NETWORK_ERROR` (or `PASSWORD_ERROR` if we can figure it out) |
+| `"Duplicate_Network"` | After `0xD004` is set, then `0xD005` will always notify as `NETWORK_ERROR`                                               |
```

Comment:
can we also add that if the password is a number eg `60` then the connection will wait 60 seconds before continuing on 

this way we can test timeouts

Reply:
Do you want this on the "Happy Path" network or a new "Timeout" network that has a modifiable connection time?

Reply:
I think the password field would be a good place so that we can test the timeout for various different paths.  but I do't think that is required, 

A `Timeout` network would work well